tests/js/handler/includes.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+// The base classes we are testing
+var express = require("../../../index.js");
```

Comment:
See above notes about this material - your goal should be to write includes such that the next line reads

gpii.express.loadTestingSupport()